### PR TITLE
RGW Multisite Support

### PR DESCRIPTION
**Pull request by jsaintrocc**
This adds support to setup RGW Multisite zones.  It replicates this procedure http://docs.ceph.com/docs/jewel/radosgw/multisite/.

### CODE REVIEW

File: roles/ceph-rgw/tasks/main.yml [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
   when: radosgw_keystone
+  # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
+  static: False
 
 - include: start_radosgw.yml
   when: not rgw_containerized_deployment
+  # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
+  static: False
+
+- name: Include rgw multisite playbooks
```
Comment (leseb) [inline roles/ceph-rgw/tasks/main.yml]: for consistency purpose, please remove all the capitals after `name:`
Thanks:

**Comment by leseb** [comment]
I'd like to merge https://github.com/ceph/ceph-ansible/pull/939/ before this one, once it's done, could you change the location of `purge-multisite.yml` and `rgw-standalone.yml` in the `infrastructure-playbooks` directory?

**Comment by jsaintrocc** [comment]
Doing it now :-)

**Comment by leseb** [comment]
Thanks @jsaintrocc, one last thing, I'm curious about the idempotency of these tasks, we are using the command module a lot. Are all the Ceph commands idempotent? Can we run this scenario multiple times without failure?

**Comment by jsaintrocc** [comment]
Idempotency is taken care of in /Users/jsaint201/ceph-ansible-master/roles/ceph-rgw/tasks/multisite/checks.yml.  All the command tasks check for existence before running.  I've run it about a million times so I can confirm that it's idempotent ;-).

**Comment by leseb** [comment]
@jsaintrocc haha ok for million times :)
Let's go with this one then.

Really nice patch btw, thanks!

**Pull request merged**